transactions/attempt_withdraw_NFT_exploit.cdc

```diff
@@ -0,0 +1,22 @@
+import DAOTreasury from "../contracts/DAOTreasury.cdc"
+transaction() {
+  
+  prepare(signer: AuthAccount) {
+    let COLLECTION_ID =  "A.f8d6e0586b0a20c7.ExampleNFT.Collection"
+    
+    let treasury <- signer.load<@DAOTreasury.Treasury>(from: DAOTreasury.TreasuryStoragePath)
+        ?? panic("A DAOTreasury doesn't exist here.")
+
+    let collection = treasury.borrowCollection(identifier: COLLECTION_ID)
+    let nft <- treasury.withdrawNFT(identifier: COLLECTION_ID, id:0)
```

Comment:
same thing here -- move everything below `borrowCollection` into `execute`!

Reply:
following up -- i think this is actually fine because we need to do a signer.save at the end, so lets just leave everything in prepare :)